go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -515,7 +526,7 @@
         expect { updater.updated_go_sum_content }.
           to raise_error(error_class) do |error|
           expect(error.message).to include(
-            "go: github.com/deislabs/oras@v0.10.0 requires\n"\
+            "go: github.com/deislabs/oras@v0.9.0 requires\n"\
```

Comment:
This is confusing, but as best I can tell the error message is reported slightly differently by `go get`.

I'd feel a bit more comfortable if someone else can take a look at it too.

Reply:
Yep, this is def just an error message getting reported differently, because the same change was necessary over in https://github.com/dependabot/dependabot-core/pull/3733/files#diff-983a4baf78264a5fd352fd21d2ba73a5a6a594bbb6f7bee2c912c56f10df772cR518